tv_apps/smart-system/js/app_modal_dialog.js

```diff
@@ -20,6 +20,10 @@
   exports.AppModalDialog = function AppModalDialog(app) {
     this.app = app;
     this.containerElement = app.element;
+    this.smartBubble = document.createElement('smart-bubbles');
+    this.smartBubble.addEventListener('all-items-bubbled', function() {
```

Comment:
The same problem, shouldn't we append dom elements that we want to bubble _inside_ `smart-bubbles`?